nix/offchain.nix

```diff
@@ -190,6 +190,18 @@ in
             };
           };
 
+          # Inorder to use as a spago module, it most be a store path to the
+          # spago module repository, and must have git revison that is for
+          # ensuring version.
+          spagoModule = lib.mkOptionType {
+            name = "spagoModule";
+            description = "noun";
```

Comment:
Needs a description

Reply:
https://github.com/NixOS/nixpkgs/blob/7b2a482ea0240ef989001c6a6c28b47db7790483/lib/types.nix#L113

`noun` is a proper description

Reply:
Ah, well I think this is supposed to _describe_ what the option does, as opposed to be a specific tag. Perhaps we haven't followed the guidelines properly for this, but if you look around, you'll see that we use a specific format for descriptions, and they all tag using `Added in:`. This description actually doesn't ever get used anywhere. It's basically just documentation atm.

Reply:
Ah, okay. Thanks